src/app/shared/components/no-result-card/no-result-card.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-no-result-card',
+  templateUrl: './no-result-card.component.html',
+  styleUrls: ['./no-result-card.component.scss']
+})
+export class NoResultCardComponent implements OnInit {
```

Comment:
JFYI you may use inline template if the component has no logic and styles and is pretty short